astroid/brain/brain_fstrings.py

```diff
@@ -19,7 +19,13 @@ def _clone_node_with_lineno(
     cls = node.__class__
     other_fields = node._other_fields
     _astroid_fields = node._astroid_fields
-    init_params = {"lineno": lineno, "col_offset": node.col_offset, "parent": parent}
+    init_params = {
```

Comment:
I think we should cover this line even if it wasn't covered before, it's easy to make mistakes with this kind of modification.

Reply:
I don't really have a good idea how to meaningfully test this. It would just be a `Am I indeed passing the arguments that I'm adding in this PR to this dict`. Doesn't seem that useful.